src/pages/settings/system-settings.tsx

```
@@ -170,6 +171,73 @@ export const SystemSettingsTab = ({
+                }}
+              />
+            )}
+          />
+        </div>
+      </div>
+      <div className="flex w-full items-center justify-between">
+        <div className="mr-6 space-y-1">
+          <div className="text-medium font-medium">桌面歌词颜色</div>
+          <div className="text-sm text-zinc-500">设置桌面歌词的显示颜色</div>
+        </div>
+        <div className="w-[360px] flex justify-end">
```

ESLint

[error] Replace `w-[360px]·flex` with `flex·w-[360px]`

(prettier/prettier)